src/library/dao/trackdao.cpp

```diff
@@ -72,6 +82,36 @@ TrackDAO::TrackDAO(CueDAO& cueDao,
           m_trackLocationIdColumn(UndefinedRecordIndex),
           m_queryLibraryIdColumn(UndefinedRecordIndex),
           m_queryLibraryMixxxDeletedColumn(UndefinedRecordIndex) {
+    connect(&m_playlistDao,
```

Comment:
I think we should use here one extra signal.
The playlist dao has the knowledge what is a SET_LOG and controls the deletion process.
The track dao only provides the last played column without any business logic.

Reply:
New signal `tracksRemovedFromPlayedHistory` that only considers SET_LOG playlists.